Insights.xcodeproj/project.pbxproj

```diff
@@ -478,9 +479,11 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = "1.0.0-beta02";
+				MARKETING_VERSION = "1.0.0-beta03";
```

Comment:
update Marketing version to 1.0.0. 1.0.0-betaXX is not compatible with Apple Store semantic versioning. Also update the fastfile to remove the script to update MARKETING_VERSION automatically.

Reply:
Same for iOS Core SDK and iOS UI SDK

Reply:
👍